skrub/_drop_column_if_null.py

```diff
@@ -0,0 +1,43 @@
+# drop columns that contain all null values
+from sklearn.utils.validation import check_is_fitted
+
+from . import _dataframe as sbd
+from ._on_each_column import SingleColumnTransformer
+
+__all__ = ["DropColumnIfNull"]
+
+
+class DropColumnIfNull(SingleColumnTransformer):
+    """Drop a single column if it contains only Null, NaN, or a mixture of null
+    values. If at least one non-null value is found, the column is kept."""
+
+    def fit_transform(self, column, y=None):
+        """Fit the encoder and transform a column.
+
+        Args:
```

Comment:
can you check the formatting of the other estimators in the library, here the section header would look like 
```
Parameters
----------
```

not super important here as it is a private class, but if we decide to make it public it will be important because sphinx relies on this formatting to produce the reference html documentation